### DISCUSSION TITLE
blog doesn't appears?

### DISCUSSION THREAD

**Discussion by Geezt435** [arthelokyo/astrowind#525] [Q&A]
I followed all the tutorials in the README.md and it still looks like this, no contents found even though the content post folder has .md and mdx files:

![Capture](https://github.com/user-attachments/assets/b2d8e815-e202-400d-a2eb-420a11afed86)

**Comment by prototypa** [comment]
Hi @Geezt435 

You have to give more details but it doesn't seem to be a problem with **AstroWind** since the demo is generated directly from the build of the repository code: https://astrowind.vercel.app/blog

Let's turn this issue into a discussion post.
